geode-core/src/distributedTest/java/org/apache/geode/management/ManagementTestRule.java

```diff
@@ -238,7 +238,7 @@ public void stopManager(final VM managerVM) {
     return allMembers;
   }
 
-  public Set<DistributedMember> getAllNormalMembers() {
+  public Set<? extends DistributedMember> getAllNormalMembers() {
```

Comment:
This should probably just be a Set of InternalDistributedMember, since that seems to be what it always returns. In general you should avoid wildcards in return types - https://docs.oracle.com/javase/tutorial/java/generics/wildcardGuidelines.html

Reply:
Done. Thanks for the link, I hadn't thought about it that much.